dyno/panda/dyno_panel.py

```diff
@@ -110,32 +110,35 @@ def map(x, in_min, in_max, out_min, out_max):
     return int((x-in_min) * (out_max-out_min) / (in_max-in_min) + out_min)
 #---------------------
 
+#Run experiments
 def push_config(event):
+    run = open("experiment_main.txt")
     magnet_config = "magnets="
-    for i in range(num_magnets):
-        pwm = int(magnet_tkvars[i].get())
-        if pwm < 0:
+    for line in run:
+        for i in range (num_magnets):
+          pwm = line[i]
+          if pwm < 0:
             pwm = 0
-        elif pwm > 9:
+           elif pwm > 9:
```

Comment:
Indentation looks a bit off to me here, everything else looks great

Reply:
Thank you for pointing that out! It's been fixed